user/languages/csharp.md

```diff
@@ -73,46 +75,46 @@ mono:
 
 You can choose from the following Mono versions:
 
-| Version          | Installed Packages (linux only, OSX always includes everything)  |
+| Version          | Installed Packages (Linux only, OSX always includes everything)  |
 | ---------------- | ---------------------------------------------------------------- |
 | 3.10.0 and later | mono-complete, mono-vbnc, fsharp, nuget, referenceassemblies-pcl |
 | 3.8.0            | mono-complete, mono-vbnc, fsharp, nuget                          |
 | 3.2.8            | mono-complete, mono-vbnc, fsharp                                 |
 | 2.10.8           | mono-complete, mono-vbnc                                         |
+| none             |                                                                  |
```

Comment:
you can turn off Mono installation with this (e.g. if you only want .NET Core). I added text to clarify.